File: PHP/Cap05/c05ex04.php
<?php
// c05ex04.php

$RESP = "S";
while (!(strtoupper($RESP) != "S")) {

  printf("Entre um valor numerico: ");
  $N = intval(trim(fgets(STDIN)));

  $R = $N * 3;

  printf("Resultado = %d\n", $R);

  printf("Deseja continuar? (S/N): ");
  fflush(STDOUT);
  $RESP = strtoupper(trim(fgets(STDIN)));

  printf("\n");
}

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>

File: SCALA/Cap05/c05ex04.scala
// c05ex04.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var RESP: Char = 'S'
    var N, R: Int = 0

    while (!(Character.toUpperCase(RESP) != 'S')) {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")

      print("Deseja continuar? (S/N): ")
      RESP = Character.toUpperCase(readChar())

      println()
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}
